build/jenkins_e2e.sh

```diff
@@ -31,8 +31,11 @@ if [[ ! -z "$(git diff --name-only pages)" ]]; then
   exit 1
 fi
 
-make
-go build -tags test github.com/google/cadvisor/integration/runner
+# Build & test with go 1.7
+docker run --rm \
```

Comment:
One option is to detect if we are in a containerized environment by setting an environment variable. If we are inside a container, then run the make command directly. If not, then invoke make run_container with the actual make command as an argument.